### PR TITLE
Style modules

### PR DESCRIPTION
**Pull request by michaelghinrichs**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,9 @@
 <HTML>
   <head>
-    <link rel="stylesheet" type="text/css" href="index.css">
+    <link rel="stylesheet" type="text/css" href="assets/css/index.css">
```
Comment (awseward) [inline index.html]: Can we do `/assets/...`?

**Comment by awseward** [comment]
+1 pending comment

File: index.html (hunk 1 above)
Comment (michaelghinrichs) [inline index.html]: nope, that does not work, however `./assets/...` does

Comment (michaelghinrichs) [inline index.html]: this is explained in better detail here http://stackoverflow.com/questions/2005079/absolute-vs-relative-urls

**Pull request merged**